Source/JavaScriptCore/heap/MarkedSpace.cpp

```diff
@@ -236,8 +236,8 @@ void MarkedSpace::sweepPreciseAllocations()
         if (allocation->isEmpty()) {
             if (auto* set = preciseAllocationSet())
                 set->remove(allocation->cell());
-            if (allocation->isLowerTier())
-                static_cast<IsoSubspace*>(allocation->subspace())->sweepLowerTierCell(allocation);
+            if (allocation->isPreciseTier())
+                static_cast<IsoSubspace*>(allocation->subspace())->sweepPreciseTierCell(allocation);
```

Comment:
`sweepPreciseTierCell` is called only for PreciseTier, but `sweepPreciseTierCell` is handling `m_preciseOnly`